x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list.tsx

```diff
@@ -97,7 +97,7 @@ export const CreateEngineButton: React.FC<CreateEngineButtonProps> = ({ disabled
           <EuiText size="s">
             <FormattedMessage
               id="xpack.enterpriseSearch.content.searchApplications.createEngineTechnicalPreviewPopover.body"
-              defaultMessage="This functionality is in technical preview and may be changed or removed completely in a future release. Elastic will take a best effort approach to fix any issues, but features in technical preview are not subject to the support SLA of official GA features."
+              defaultMessage="This functionality may be changed or removed completely in a future release. Elastic will take a best effort approach to fix any issues, but features in technical preview are not subject to the support SLA of official GA features."
```

Comment:
The second sentence can be nixxed completely! So the entire message is

> This functionality may be changed or removed completely in a future release.

![CleanShot 2023-04-28 at 09 12 32](https://user-images.githubusercontent.com/4490333/235157041-e84db4ea-da96-407e-a19f-f83c20bb74c1.png)

Thank you 100x @kderusso!

Reply:
(This is because the full message is within the flyout, so they'll be forewarned once they get there)